.github/workflows/nightly_check.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+          cache: pip
```

Comment:
Can this option be shared everywhere?

Reply:
Not 100% sure but I want to check that works with this workflow.